src/formatter/formatComplexDataStructure.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+
+import collapse from 'collapse-white-space';
```

Comment:
Yes `complexDataStrucure` is for Object and Array. Things that need exploration to be converted. After reading the jest core, [they use `printComplexValue` naming](https://github.com/facebook/jest/blob/master/packages/pretty-format/src/index.js#L878) for this use case